pkgs/development/python-modules/pytest-profiling/default.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  gprof2dot,
+  pypaInstallHook,
+  pytest,
+  pytest-virtualenv,
+  pytestCheckHook,
+  setuptoolsBuildHook,
+  six,
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-profiling";
+  version = "1.8.0";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "man-group";
+    repo = "pytest-plugins";
+    rev = "refs/tags/${version}";
+    hash = "sha256-OYPdfMZrG4aTVnSAorVYxBgNNxIqHhO5yF8PUGhPBgA=";
+  };
+
+  sourceRoot = "${src.name}/pytest-profiling";
+
+  # Python has some trouble finding `common_setup.py` in the root
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-fail 'sys.path.append(os.path.dirname(os.path.dirname(__file__)))' 'sys.path.append("${src}")'
+  '';
+
+  nativeBuildInputs = [
+    pypaInstallHook
+    setuptoolsBuildHook
+  ];
+
+  dependencies = [
+    gprof2dot
+    pytest
+    six
+  ];
+
+  nativeCheckInputs = [
+    pytest-virtualenv
+    pytestCheckHook
+  ];
+
+  # ValueError: 'int' is not callable
+  doCheck = false;
```

Comment:
You have both `disabledTests = [ ... ]` and `doCheck = false`

Reply:
Yes, as I had some tests fail while working on this, but eventually ran into this issue when first starting `pytest`. If we can solve the latter the former is likely still be a problem, so it's nice to still have `disabledTests`